server/services/withings/lib/index.js

```diff
@@ -0,0 +1,37 @@
+// COMMANDS
+const { init } = require('./withings.init');
+const { saveVar } = require('./withings.saveVar');
+const { deleteVar } = require('./withings.deleteVar');
+const { deleteDevices } = require('./withings.deleteDevices');
+const { getServiceId } = require('./withings.getServiceId');
+const { poll } = require('./withings.poll');
+
+/**
+ * @param {Object} gladys - The gladys object.
+ * @param {string} serviceId - Identification of the service.
+ * @param {string} withingsServerUrl - Withings server url.
+ * @param {string} integrationName - [Optional] By default interationName='withings', param used only for test.
+ * @description Create all device if not exist by listening
+ * @example
+ * const kodiManager = KodiManager(gladys, serviceId)
```

Comment:
Maybe WithingsHandler instead of kodiManager ? :)

Reply:
yes, i fix it